### PR TITLE
Beta to Stage

### PR DESCRIPTION
**Pull request by ravinderwalkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ on:
     branches: [ giddh-2.0, beta-stage, beta-branch, production ]
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
-concurrency: stage-build
+concurrency: beta-build
```
Comment (prateek-giddh) [inline .github/workflows/build.yml:11]: Do revert this

File: angular.json [hunk 2]
```diff
@@ -213,7 +212,7 @@
                                     "externals": "replace"
                                 }
                             },
-                            "baseHref": "https://app.giddh.com/",
+                            "baseHref": "https://beta.giddh.com/",
```
Comment (prateek-giddh) [inline angular.json:215]: Do revert this

File: apps/web-giddh/src/app/actions/login.action.ts [hunk 3]
```diff
@@ -316,7 +316,7 @@ export class LoginActions {
             ofType(LoginActions.LogOut),
             map((action: CustomActions) => {
                 if (PRODUCTION_ENV && !(isElectron || isCordova)) {
-                    window.location.href = 'https://stage.giddh.com/login/';
+                    window.location.href = 'https://beta.giddh.com/login/';
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/actions/login.action.ts:319]: Do revert this

File: apps/web-giddh/src/app/app.component.ts [hunk 4]
```diff
@@ -79,7 +79,7 @@ export class AppComponent implements AfterViewInit, OnInit, OnDestroy {
         if (!(this._generalService.user && this._generalService.sessionId)) {
             if (!window.location.href.includes('login') && !window.location.href.includes('token-verify') && !window.location.href.includes('download')) {
                 if (PRODUCTION_ENV && !(isElectron || this.isCordova)) {
-                    window.location.href = 'https://stage.giddh.com/login/';
+                    window.location.href = 'https://beta.giddh.com/login/';
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/app.component.ts:82]: Do revert this

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 5]
```diff
@@ -730,7 +730,7 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
                 if (isElectron) {
                     this.router.navigate(['/login']);
                 } else {
-                    window.location.href = (environment.production) ? `https://stage.giddh.com/login/?action=logout` : `https://test.giddh.com/login/?action=logout`;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/header/header.component.ts:733]: Do revert this

File: apps/web-giddh/src/assets/magic-link-assets/magic-link-script.js [hunk 6]
```diff
@@ -374,7 +374,7 @@ var app = new Vue({
                     apiBaseUrl = 'https://apidev.giddh.com/';
                     break;
                 case 'stage.giddh.com':
-                    apiBaseUrl = 'https://apirelease.giddh.com/';
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/magic-link-assets/magic-link-script.js:377]: Do revert this

**Review by prateek-giddh — CHANGES_REQUESTED** [review]
Please look into all the env. variables that are getting overridden due to BETA code

**Review by prateek-giddh — APPROVED** [review]

**Review by aditi-tiwari17 — APPROVED** [review]

**Pull request merged**